### PR TITLE
[CARBONDATA-802]Select query is throwing exception when newly column is added without any default value

### PR DESCRIPTION
**Pull request by nareshpr**
If user does not provide default value in add column, dictionary & sort index file will not be created which is throwing exception while querying. Current change will create dictionary file with default null value if not default value is provided in add column query

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1266/

File: integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala [hunk 1]
```diff
@@ -229,31 +229,37 @@ class AlterTableProcessor(
...
-                col(0),
-                tableIdentifier,
-                storePath,
-                elem._2)
+
+    newCols.foreach { col =>
+      var rawData: String = null
+      for (elem <- alterTableModel.tableProperties) {
+        if (elem._1.toLowerCase.startsWith("default.value.")) {
+          if (col.getColumnName.equalsIgnoreCase(elem._1.substring(14))) {
```
Comment (gvramana) [inline integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala]: create and a val for "default.value." and use length function.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1273/

**Comment by nareshpr** [comment]
Fixed review comments. Pls review and merge

**Comment by gvramana** [comment]
LGTM

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1282/

**Pull request merged**